### DISCUSSION TITLE
Open in new window

### DISCUSSION THREAD

**Discussion by hayian** [datalust/seq-tickets#1112] [Ideas]
When looking through a list we'll often want to dive deeper into a httprequestid to see specifics on that request, so we'll usually open a new window, and then copy and paste the requestid into the search box to look further into that.  It'd be nice to be able to click on the parameter, then right click on find and open in new tab, or at least something that achieves the same without losing the list you're currently looking through.

**Comment by andymac4182** [comment]
It would be great if middle click worked on the current menu options. I use that on websites all the time and forget that Seq doesn't work nicely with it.

**Comment by nblumhardt** [comment]
Thanks for the feedback, will take a look at this one :+1:

**Comment by abagonhishead** [comment]
Any movement on this? It would be super useful.

Currently if I want to filter down to e.g. a single trace ID from a log event while keeping my existing query open, I have to click the tick and 'find just this' then ctrl + click/middle click the back button to open my previous query in a new tab. It's a bit of a faff and presumably puts extra load on the server since I'm essentially refreshing the query.

**Reply by nblumhardt** [reply]
Thanks for the feedback - still on the radar, nothing new to report just yet, unfortunately.

**Comment by bencullpinch** [comment]
Hi Nick, just wanted to put in a feature request while I was in amongst the discussions today, and this would be my biggest daily life improver. Mostly driven by the Find menu as per (https://github.com/datalust/seq-tickets/discussions/1157) as well.
